starwars/src/App.js

```diff
@@ -17,16 +18,24 @@ class App extends Component {
         return res.json();
       })
       .then(data => {
-        this.setState({ starwarsChars: data.results });
+        this.setState({ starwarsChars: data.results }, () =>  {})//console.log('sw:',this.state.starwarsChars[0]));
```

Comment:
Good use of debugger here. I'd also make sure you get a handle on the React Dev Tools and use those to peek at your objects available.